Pipfile

```diff
@@ -4,9 +4,13 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-astroid = ">=2.4.2,<2.5.4"
+astroid = "*"
 autopep8 = "*"
 bandit = "*"
+mypy = "*"
+pylint = "==2.12.2"
```

Comment:
lets open an issue for upgrading pylint and add a comment here to direct to the issue